conf/messages.cy

```diff
@@ -113,6 +113,21 @@ cf.error.not-found.message.go-to-home-page-text1=Neu gallwch
 cf.accounts.view-customs-account=fwrw golwg dros eich cyfrifon ariannol ar gyfer y tollau
 
 
+# Undeliverable Email Address
+# ----------------------------------------------------------
+cf.undeliverable.email.title=Mae problem gyda’ch cyfeiriad e-bost ar gyfer y Gwasanaeth Datganiadau Tollau
+cf.undeliverable.email.heading=Mae problem gyda’ch cyfeiriad e-bost ar gyfer y Gwasanaeth Datganiadau Tollau
+cf.undeliverable.email.p1=Gwnaethom geisio anfon e-bost atoch ond nid oedd modd ei ddosbarthu.
+cf.undeliverable.email.p2=Eich e-bost cofrestredig yw <strong class="bold">{0}</strong>.
+
+cf.undeliverable.email.verify.heading=Cadarnhau’ch cyfeiriad e-bost
+cf.undeliverable.email.verify.text.p1=Os mae’r cyfeiriad e-bost hwn yn gywir, bydd angen i chi ei gadarnhau er mwyn parhau i ddefnyddio Gwasanaeth Datganiadau Tollau.
```

Comment:
I noticed your double '' in the English version as I assume the single ' doesn't display? Do we need to double them in the welsh as well?

Reply:
@AdamCoops Single quotes used in English messages file and welsh file are different. Welsh translation is working with a single quote in the messages file.
<img width="1157" alt="image" src="https://github.com/hmrc/customs-cash-account-frontend/assets/29434052/d35ee7bf-1ecb-4654-b927-4703e28075db">

Reply:
double quotes are required in Html classes